HW04/aleksei_zhuchkov/lib/mentor.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class Mentor
+  attr_accessor :students_list, :notifications_list
+  attr_reader :name, :surname
+
+  def initialize(name:, surname:)
+    @name = name.capitalize
```

Comment:
You don't really need this method (capitalize), but you can leave it

Reply:
Hi, I've done it, you can check it in this commit https://github.com/labs-ruby/homeworks-2022/pull/96/commits/7a48c4d003421b1f3ba66d6d41e8a4137699535e